website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,255 @@
+15:10:14  3 of 28 START seed file main.raw_payments ...................................... [RUN]
+....
+
+15:10:15  27 of 28 PASS relationships_orders_customer_id__customer_id__ref_customers_ .... [PASS in 0.32s]
+15:10:15  
+15:10:15  Finished running 3 seeds, 3 view models, 20 data tests, 2 table models in 0 hours 0 minutes and 1.52 seconds (1.52s).
+15:10:15  
+15:10:15  Completed successfully
+15:10:15  
+15:10:15  Done. PASS=28 WARN=0 ERROR=0 SKIP=0 TOTAL=28
+
+```
+
+For complete details, refer to the [dbt command reference](/reference/dbt-commands).
+
+:::note
+
+The steps will fail if you decide to run this project in your data warehouse (outside of this DuckDB demo). You will need to reconfigure the project files for your warehouse. Definitely consider this if you are using a community-contributed adapter.
+
+:::
+
+
+### Troubleshoot
+
+    <Expandable alt_header="Could not set lock on file error" >
+
+    ```Jinja
+
+    IO Error: Could not set lock on file "jaffle_shop.duckdb": Resource temporarily unavailable
+
+    ```
+
+    This is a known issue in DuckDB. Try disconnecting from any sessions that are locking the database. If you are using DBeaver, this means shutting down DBeaver (disconnecting doesn't always work).
+
+    As a last resort, deleting the database file will get you back in action (_but_ you will lose all your data).
+
+    </Expandable>
+
+
+  </TabItem>
+ 
+  <TabItem value="web" label="Web browser">
+
+1. Go to the `jaffle-shop-template` [repository](https://github.com/dbt-labs/jaffle_shop_duckdb) after you log in to your GitHub account. 
+1. Click **Use this template** at the top of the page and choose **Create new repository**.
+1. Click **Create repository from template** when you’re done setting the options for your new repository.
+1. Click **Code** (at the top of the new repository’s page). Under the **Codespaces** tab,  choose **Create codespace on main**. Depending on how you've configured your computer's settings, this either opens a new browser tab with the Codespace development environment with VSCode running in it or opens a new VSCode window with the codespace in it. 
+1. Wait for the codespace to finish building by waiting for the `postCreateCommand` command to complete; this can take several minutes:
+
+    <Lightbox src="/img/codespace-quickstart/postCreateCommand.png" title="Wait for postCreateCommand to complete" />
+
+    When this command completes, you can start using the codespace development environment. The terminal the command ran in will close and you will get a prompt in a brand new terminal. 
+
+1. At the terminal's prompt, you can execute any dbt command you want. For example:
+
+    ```shell
+    /workspaces/test (main) $ dbt build
+    ```
+
+    You can also use the [duckcli](https://github.com/dbcli/duckcli) to write SQL against the warehouse from the command line or build reports in the [Evidence](https://evidence.dev/) project provided in the `reports` directory.
```

Comment:
just a heads up the cli repo says it archived but i can't find anything else so maybe it's ok?